turbinia/config/turbinia_config.py

```diff
@@ -47,8 +44,10 @@
 TASK_MANAGER = 'PSQ'
 
 # File to log to; set this as None if log file is not desired
-# By default, Turbinia logs are written to a directory (GCS_MOUNT_DIR) in the GCS mount
-LOG_FILE = '%s/output/logs/turbinia.log' % GCS_MOUNT_DIR
+# By default, Turbinia logs are written to a directory (GCS_MOUNT_DIR)
+# in the GCS mount
+#LOG_FILE = '%s/output/logs/turbinia.log' % GCS_MOUNT_DIR
+LOG_FILE = '/dev/null'
```

Comment:
Not logging to GCS by default makes sense, but could we log elsewhere locally by default?

Reply:
Yeah, sure makes sense. I think that we should skip this placeholder config all together in the long run though and bundle a config file with the package. Then bail out early if it is missing.

Reply:
I'm not sure if having a separate config will gain us anything because it still won't contain enough to be a full valid config.  It should already bail out, but there is currently a workaround to set some config vars to 'None' (the string) instead of None (the object) because we now have some config vars that are optional.  Once the config has a mechanism to set optional parameters (tracked in #181), I think this will be a little better.  We could potentially move this file to somewhere else in the repo if we want to make it more obviously a template config.  All things for a separate PR though.

Reply:
Cool, yeah makes sense.